content/pt/docs/reference/access-authn-authz/authentication.md

```diff
@@ -0,0 +1,892 @@
+ 
+* Username: um valor (String) que identifica o usuário final. Valores comuns podem ser `kube-admin` ou `jane@example.com`
+* UID: um valor (String) que identifica o usuário final e tenta ser mais consistente e único do que username.
+* Groups: Um conjunto de valores em que cada item indica a associação de um usuário há uma coleção lógica de usuários. Valores comuns podem ser `system:masters` ou `devops-team`.
+* Campos extras: um mapa para uma lista que armazena informações adicionais em que autorizadores podem achar útil.
+ 
+Todos os valores são opacos para o sistema de autenticação e somente trazem significância quando interpretados por um autorizador (/docs/reference/access-authn-authz/authorization/).
+ 
+É possível habilitar múltiplos métodos de autenticação. Deve-se normalmente usar pelo menos dois métodos:
+ 
+- _Tokens_ para contas de serviço;
+- Pelo menos um outro método de autenticação para usuários.
+ 
+Quando múltiplos módulos de autenticação estão habilitados, o primeiro módulo a autenticar com sucesso uma requisição termina o fluxo de avaliação da mesma.
+ 
+O servidor de API não garante a ordem em que os autenticadores são processados.
+ 
+O grupo `system:authenticated` é incluído na lista de grupos de todos os usuários autenticados.
+ 
+Integrações com outros protocolos de autenticação, como LDAP, SAML, Kerberos, alternate x509 schemes, etc, podem ser alcançadas utilizando-se de um proxy (#authenticating-proxy) ou webhook (#webhook-token-authentication) de autenticação.
+ 
+### Certificados de cliente X509
+ 
+Autenticação via certificados de cliente pode ser habilitada ao passar a opção `--client-ca-file=ARQUIVO` para o servidor de API. O arquivo referenciado deve conter um ou mais autoridades de certificação usadas para validar o certificado de cliente passado para o servidor de API. Se o certificado de cliente é apresentado e verificado, o nome comum (CN) do sujeito é usado como o nome de usuário para a requisição. A partir da versão 1.4, certificados de cliente podem também indicar o pertencimento de um usuário a um grupo utilizando o campo de organização do certificado. Para incluir múltiplos grupos para o usuário, deve-se incluir múltiplos campos de organização no certificado.
+ 
+Por exemplo, utilizando o comando de linha `openssl` para gerar uma requisição de assinatura de certificado:
+ 
+``` bash
+openssl req -new -key jbeda.pem -out jbeda-csr.pem -subj "/CN=jbeda/O=app1/O=app2"
+```
+ 
+Isto criaria um arquivo de tipo CSR (requisição de assinatura de certificado) para o usuário "jbeda" pertencendo a dois grupos: "app1" e "app2".
+ 
+Veja como gerar um certificado de cliente em [Gerenciando Certificados] (/docs/concepts/cluster-administration/certificates/)
```

Comment:
`[Gerenciando Certificados] (/docs/concepts/cluster-administration/certificates/)` is `[Gerenciando Certificados](/docs/concepts/cluster-administration/certificates/)`

Reply:
Fixed on commit 9b5815e